models/intel/face-detection-0100/description/face-detection-0100.md

```diff
@@ -0,0 +1,53 @@
+# face-detection-0100
+
+## Use Case and High-Level Description
+
+Face detector based on MobileNetV2 as a backbone with a
+multiple SSD head for indoor/outdoor scenes shot by a front-facing camera.
+
+## Example
+
+![](./face-detection-0100.png)
+
+## Specification
+
+| Metric                                                        | Value                   |
+|---------------------------------------------------------------|-------------------------|
+| AP ([WIDER](http://mmlab.ie.cuhk.edu.hk/projects/WIDERFace/)) | 86.66%                  |
+| GFlops                                                        | 0.786                   |
+| MParams                                                       | 1.828                   |
+| Source framework                                              | PyTorch*                |
+
+Average Precision (AP) is defined as an area under the
+[precision/recall](https://en.wikipedia.org/wiki/Precision_and_recall)
+curve. All numbers were evaluated by taking into account only faces bigger than
+64 x 64 pixels.
+
+## Performance
+
+## Inputs
+
+1. name: "input" , shape: [1x3x256x256] - An input image in the format [BxCxHxW],
+   where:
+
+    - B - batch size
+    - C - number of channels
+    - H - image height
+    - W - image width
+
+   Expected color order - BGR.
+
+## Outputs
+
+1. The "boxes" is a blob with shape: [N, 5], where N is the number of detected
```

Comment:
are you test IRs and sure that they contains the same output names? (my experience of work with `person-detection-asl` says that it can be different...)

Reply:
replaced models by their variants with DetectionOutput